utils/debug.c

```diff
@@ -329,6 +329,9 @@ void print_diff_percent(uint64_t base_nsec, uint64_t pair_nsec)
 	const char *sc = TERM_COLOR_NORMAL;
 	const char *ec = TERM_COLOR_NORMAL;
 
+	if (out_color == COLOR_ON)
+		ec = TERM_COLOR_RESET;
+
 	if (base_nsec == 0) {
 		pr_out("%s%7s%s ", TERM_COLOR_RED, "N/A", ec);
```

Comment:
I think we should check COLOR_ON before using the RED color.  And below BLUE too.

Reply:
Right. I've fixed it.